contributing-snippets.md

```diff
@@ -0,0 +1,21 @@
+# Contributing to Tiltfile Snippets
```

Comment:
We don't have a regular CONTRIBUTING.md file in this repo, I wonder if this should be it and we add a little bit of boilerplate for contributing documentation fixes?

Reply:
Good point.
I do wonder though how much this will blow up the scope of this particular story (given that I'm also going to be gone for two weeks)
Here's my suggestion: For this story, leave it as is. 

Follow up afterwards in a separate PR to make this a more generic contributing md?

Reply:
Design looks simple and good, however, I would recommend treating color of 'user name' slightly different for easier readability. Would it make sense to include a 'date' stamp for when the snippet was submitted?

Just to recap, for each snippet the following information will be included:

- Title
- 1 to 2 line Context
- Actual snippet
- Copy icon
- Permalink
- Link to the docs (described better)
- _(maybe)_ Submitted by `user name` 
- _(maybe)_ Tags

Reply:
Personally, I don't feel like a submittal date would add any relevant information and risk visually overloading the design.